skfem/mesh/mesh.py

```diff
@@ -418,7 +419,8 @@ def __post_init__(self):
             self.t = np.ascontiguousarray(self.t)
 
         # run validation
-        self.is_valid(debug=False)
+        if validate:
+            self.is_valid()
```

Comment:
By omitting debug=False, I think you are causing the time consuming checks to run always. Maybe get rid of debug kwarg and move log level check here?

Reply:
I added a logger.DEBUG msg any time is_valid gets called (this should probably stay to alert you that something expensive happened if you forgot debug was on). It seems like is_valid is getting called at least twice per mesh creation.

The traceback is just temporary while I try to figure out where these calls to `is_valid` are coming from.

Reply:
I also added a `raise_` keyworkd to `is_valid`, which is occasionally useful to halt and get the stack trace when troubleshooting

Reply:
Ah, there's a temp mesh in `from_meshio`

https://github.com/kinnala/scikit-fem/blob/926c165a21cacad65855a0c4e15409bf84763af7/skfem/io/meshio.py#L111-L112

Probably only should validate the actual mesh returned.